force/api.go

```diff
@@ -27,6 +28,7 @@ type ForceApi struct {
 	apiMaxBatchSize        int64
 	logger                 ForceApiLogger
 	logPrefix              string
+	httpClient             *http.Client
```

Comment:
I'm a little confused. This is a struct.. so it won't accept a client that's not http.Client
I think what you want is just an interface that does most Client-y things?

Reply:
golang doesn't expose an http.Client interface. I could create one that encapsulates the subset required by go-force; that's what I've done in sfsync. Here it seemed simpler to just use the standard http.Client since tests have to go out to the internet anyway. * shrug *, happy to use an interface. Will do.